### DISCUSSION TITLE
SwiftUI UIViewRepresentable for IOSVideoPlayerView

### DISCUSSION THREAD

**Discussion by mazz** [kingslay/KSPlayer#732] [General]
Does a SwiftUI View wrapper in the form of a UIViewRepresentable already exist in the KSPlayer codebase for IOSVideoPlayerView or would I need to make one?

**Comment by kingslay** [comment]
你可以用KSVideoPlayerView

**Reply by mazz** [reply]
KSVideoPlayerView is great but there are a few things I have issue with:

1) No autorotate to landscape fullscreen by default, not clear to me how to set this.

2) PiP loses its connection with the original View hierarchy after restoring to the original video. When the user taps PiP -> Restore to full size, the original video in the original view hierarchy gets lost(video player is black).

3) Not clear how to remove the built-in Close (X) button.

**Comment by kingslay** [comment]
那你可以简单的对IOSVideoPlayerView进行包装下，这样就可以了。

**Reply by mazz** [reply]
Thanks for the suggestion. I will go with IOSVideoPlayerView.

Translator: 感谢您的建议。我将使用 IOSVideoPlayerView
